docs/admin/logical-replication.rst

```diff
@@ -30,6 +30,9 @@ Logical replication is useful for the following use cases:
 - Ensure high availability if one cluster becomes unavailable.
 
 - Replicating between different compatible versions of CrateDB.
+  Please note, that replicating from a higher version to a lower version
+  is not supported since old version might not be able to process data/metadata
+  of the newer version.
```

Comment:
Should we add some error on a subscribers side? 

Unfortunately, we cannot access Version of the publisher table without connecting to the remote cluster. It means we can end up with a not working subscription. 

If we decide to show an error, ideally we should drop subscription as well (and log a message explaining why).

Reply:
Yes please, lets show an error and at best, throw the error while creating a subscription.

Reply:
Added a check which throws an error on subscription creation.